cluster/src/test/java/org/apache/iotdb/cluster/server/member/MemberTest.java

```diff
@@ -149,6 +156,7 @@ public void setUp() throws Exception {
 
     testMetaMember.setPartitionTable(partitionTable);
     MetaPuller.getInstance().init(testMetaMember);
+    logger.warn("Member test set up");
```

Comment:
Why use the warn log here?

Reply:
It was for debugging, so I will remove it.